_socketsGenerator/createAccount.js

```diff
@@ -29,11 +29,16 @@ function createAccount() {
           password: credentials.password
         };
         fs.writeFileSync('./test/newAccountInfo.json', JSON.stringify(accountInfo, null, 2), function (err) { });
+          console.log('Account was created')
       });
   });
 }
 if (!fs.existsSync(dir)){
   fs.mkdirSync(dir);
 }
-createAccount();
-console.log('Test account created');
+
+if (process.argv[2] !== 'test/main_test.js') {
```

Comment:
it'd be better to check if file with credentials exists instead of comparing args